README.md

```diff
@@ -151,6 +155,10 @@ go run ./cli test \
 
 ## Populating DApp
 
+> [!WARNING]
```

Comment:
The populate script doesn't work because it uses `-a` flag. I will replace with `-f` flag and everything seems to work

Reply:
It also doesn't work because it is now using `cast` which limits the size of arguments to function calls. In our case, this limits the size of the input, so we can't send some bounties such as SQLite.